src/Backends/Cubics/GeneralizedCubic.h

```diff
@@ -123,10 +123,18 @@ class AbstractCubic
     void set_C_MC(double c1, double c2, double c3){
         alpha[0].reset(new MathiasCopemanAlphaFunction(a0_ii(0),c1,c2,c3,T_r/Tc[0]));
     }
+    /// Set the three Mathias-Copeman constants in one shot for the component i of a mixturee
```

Comment:
Maybe just replace the other set_C_MC function?  Seems a bit redundant having two functions basically doing exactly the same thing.  Or you make the old one call the new one with index of 0